frontend/src/pages/agency-profile-form/AgencyProfileForm.js

```diff
@@ -55,53 +56,92 @@ class AgencyProfileForm extends Component {
         fileAudit: "",
         monitored: "",
         foodSafetyCertification: "",
-        "distributionDay.monday": false,
-        "distributionDay.tuesday": false,
-        "distributionDay.wednesday": false,
-        "distributionDay.thursday": false,
-        "distributionDay.friday": false,
-        "distributionDay.saturday": false,
-        "distributionDay.sunday": false,
+        mainSitePhoneNumber: "",
+        monday: false,
+        tuesday: false,
+        wednesday: false,
+        thursday: false,
+        friday: false,
```

Comment:
Why were all of these state names updated?

Reply:
See below! (-: